src/test/java/com/epam/tc/hw3/testcases/Exercise1.java

```diff
@@ -0,0 +1,170 @@
+package com.epam.tc.hw3.testcases;
+
+import com.epam.tc.hw3.library.pages.Header;
+import com.epam.tc.hw3.library.pages.HomePage;
+import com.epam.tc.hw3.library.pages.SideBar;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Exercise1 extends BaseTest {
+
+
+    //1. Open test site by URL - in parent class
+
+    @Test(priority = 0)
+    public void assertTitle() {
+        //2. Assert Browser title
+        SoftAssertions softly = new SoftAssertions();
+        String browserTitle = testDriver.getTitle();
+        softly.assertThat(browserTitle).as("Incorrect browser title").isEqualTo("Home Page");
+        softly.assertAll();
+    }
+
+    @Test(priority = 1)
+    public void assertLogin() {
+        SoftAssertions softly = new SoftAssertions();
+        Header header = new Header(testDriver);
+
+        //3. Perform login
+        header.login();
+        softly.assertThat(testDriver.getCurrentUrl())
+                .as("Incorrect page opened")
+                .isEqualTo("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //4. Assert Username is loggined
+        softly.assertThat(header.getUserNameText())
+                .as("Incorrect user name")
+                .isEqualTo("ROMAN IOVLEV");
+
```

Comment:
7.	User data should be stored in the java properties file

Reply:
Fixed